core/contracts/oracle/implementation/DesignatedVoting.sol

```diff
@@ -37,7 +36,7 @@ contract DesignatedVoting is Withdrawable {
     constructor(address finderAddress, address ownerAddress, address voterAddress) public {
         _createExclusiveRole(uint(Roles.Owner), uint(Roles.Owner), ownerAddress);
         _createExclusiveRole(uint(Roles.Voter), uint(Roles.Owner), voterAddress);
-        setWithdrawRole(uint(Roles.Owner));
+        _setWithdrawRole(uint(Roles.Owner));
```

Comment:
This is now consistent with the other two calls to `_createExclusiveRole` because `Withdrawable.sol` is abstract and so should be preceeded by an underscore for internal & private functions.